airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/JdbcDatabase.kt

```diff
@@ -111,7 +111,7 @@ abstract class JdbcDatabase(protected val sourceOperations: JdbcCompatibleSource
      * close the returned stream to release the database connection. Otherwise, there will be a
      * connection leak.
      *
-     * @param statementCreator create a [PreparedStatement] from a [Connection].
+     * @paramstatementCreator create a [PreparedStatement] from a [Connection].
```

Comment:
nit: revert this

Reply:
Formatter did this 🤷‍♂️ 
I don't quite know why